web/pages/meetings/aps/62nd_dpp_mini_conf_followups.md

```diff
@@ -0,0 +1,23 @@
+title: 62nd APS DPP - 11.04 Mini-Conference Followups
+author: Erik T. Everson
+hidetitle: True
+
+[Zoom link]: https://harvard.zoom.us/j/91600794594?pwd=L09iTGtTRUN1RmpsVnNvU05LRnNwQT09
+[chat room]: https://app.element.io/#/room/#plasmapy:openastronomy.org
+
+# 62nd APS DPP - 11.04 Mini-Conference | Post-Meeting Discussions
+### Growing an open-source software ecosystem for plasma science
+
+<div style="height: 12px"><!-- Adding vertical whitespace --></div>
+
+**Where:** On Zoom [link will be updated] <br/>
```

Comment:
```suggestion
**Where:** [Zoom link] <br/>
```
We could add the Zoom link in now.  (Hopefully I'm remembering how to use Markdown links right...)